src/blockchain_db/blockchain_db.cpp

```diff
@@ -359,12 +376,6 @@ void BlockchainDB::show_stats()
 
 void BlockchainDB::fixup()
 {
-  if (is_read_only()) {
-    LOG_PRINT_L1("Database is opened read only - skipping fixup check");
-    return;
-  }
-
-  set_batch_transactions(true);
```

Comment:
We removed this in an earlier commit but reverted (https://github.com/loki-project/loki/commit/de7235e146ff963e68258ae5df073d4eee964a21) because this needs to be set true on start up, but wasn't being set anywhere else at startup other than the fixup() function so the daemon wouldn't run and complain it wasn't set.

We should of moved this call out to the caller of this function in blockchain::init().

Reply:
Yeah, I saw that and thought I chose that change when merging but apparently not.  Will go ahead and take care of that.